### DISCUSSION TITLE
Which day is dropped from demand data leap years?

### DISCUSSION THREAD

**Discussion by gschivley** [NREL/ReEDS-2.0#181] [Q&A: Data] [answered]
I assumed that Feb 29 would be dropped from demand data in 2008/12, but [the code in eer_to_reeds.py](https://github.com/NREL/ReEDS-2.0/blob/9b159a182a95bd7ff96bb8fa2f9d1110744fb51a/hourlize/eer_to_reeds/eer_to_reeds.py#L40) appears to just take the first 8760 hours. Is that the case?

**Comment by vcelloho** [comment, marked as answer]
In a leap year the final 24 hours are dropped, this is to avoid creating a discontinuity within the hourly dataset.
